pkg/driver/driver.go

```diff
@@ -136,24 +137,30 @@ func getDBClient(server string, port string) (*etcd.DBClient, error) {
 }
 
 // Start starts the gRPC server
-func (driver *Driver) Start(nodeService bool) error {
+func (driver *Driver) Start(nodeService bool, nfsMonitor bool) error {
 	go func() {
 		driver.grpc = NewNonBlockingGRPCServer()
 		if nodeService {
 			driver.grpc.Start(driver.endpoint, driver, nil, driver)
 		} else {
 			driver.grpc.Start(driver.endpoint, driver, driver, nil)
+			if nfsMonitor {
+				driver.flavor.StartNFSMonitor()
+			}
```

Comment:
Once you change the interface to configureMonitor, this can change to StartMonitors() which will start all registered monitors.

Reply:
@rgcostea do you think its better to pull this into a separate sidecar instead?

Reply:
I'm not sure that is necessarily better as that includes the overhead of building and managing another container.

Reply:
ok.let me pull this into separate package then with above changes.